### ISSUE TITLE
Quarto PDF cannot completely preview in Viewer Pane if it includes Chinese.

### ISSUE THREAD

**Issue by GlekoMa** [rstudio/rstudio#11116]
### System details

    RStudio Edition : Desktop
    RStudio Version : 2022.02.1 Build 461
    OS Version      : Windows11
    R Version       : 4.1.3

### Steps to reproduce the problem
1. Create a Quarto Document and choose PDF
2. Delete all the original code and write:
```
---
title: "Use Quarto"
format: 
  pdf: 
    documentclass: ctexart
---

## Test

Chinese: 中文

曾经沧海难为水，除却巫山不是云。

Hello? Is any body in there?
```
3. Click the "Render" 
### Describe the problem in detail
![pic](https://user-images.githubusercontent.com/97289978/166631626-f97d0289-4612-426a-b4ac-dc30b52ded21.png)
Many Chinese words lacked in the Viewer Pane, but the PDF it created is completed.Just like:
![pic2](https://user-images.githubusercontent.com/97289978/166630534-94713ab5-c264-404a-b36d-258b6f77199c.png)
And I noticed that vscode can completely preview the PDF with extension "PDF Viewer".Just like:
![pic3](https://user-images.githubusercontent.com/97289978/166631147-38355277-ef1e-4105-b40a-8ea52a0fc622.png)
### Describe the behavior you expected
Fix the bug of uncompletely preview when the PDF include Chinese.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@GlekoMa Thank you for raising the issue! I can also reproduce something similar with R Markdown files. It's not exactly the same—no characters at all are displayed—so I'm also filing a separate issue for it (#111120) in case they're not duplicates.

**Comment by sammo3182** [comment]
Same problem here running quarto on Rstudio and Win 11

**Comment by xiaolong-y** [comment]
Same issue is replicated on Mac's RStudio when setting the compilation engine to both `pdfLaTex` and `XeLaTeX` via `sweave`.

**Comment by GlekoMa** [comment]
> Has this issue been fixed yet?

@sammo3182 

Sorry for the late check. I think this issue has been fixed. I have tried again in RStudio 2023.03.0 Build 386 (Windows) and found it run perfectly:

![image](https://user-images.githubusercontent.com/97289978/236177858-e60bd4de-f5e0-4d61-a418-03fea1a7317f.png)

> I still have the same problem on quarto 1.3.340.

I guess it has nothing to do with quarto. Try updating the RStudio. :sparkles:
